### PR TITLE
Bulk actions refinements

### PR DESCRIPTION
**Pull request by cantsin**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/738)

## What does this change?

- Bulk actions: 
  - If assigned section is changed, unset user and set status to New (same as with individual form)
  - Disable judicial district

Note: No messaging regarding "section will reset assignee and set status to new" yet -- may spin up a separate ticket to add this for both individual and bulk update forms. 

### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Re-check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/forms.py [hunk 1]
```diff
@@ -1290,7 +1290,10 @@ class BulkActionsForm(Form, ActivityStreamUpdater):
     district = ChoiceField(
         label='Judicial district',
         widget=ComplaintSelect(
-            attrs={'class': 'text-uppercase crt-dropdown__data'},
+            attrs={
+                'class': 'text-uppercase crt-dropdown__data',
+                'disabled': 'disabled'
```
Comment (Jkrzy) [inline crt_portal/cts_forms/forms.py:1295]: With this disabled, a form submission clears any existing value in `Report.district`.

File: crt_portal/cts_forms/tests/test_forms.py [hunk 2]
```diff
@@ -464,6 +464,20 @@ def test_post_with_ids(self):
             self.assertEquals(last_activity.description, 'Updated from "None" to "DELETE_USER"')
             self.assertEquals(last_activity.actor, user)
 
+    def test_post_with_section_status_and_assignee(self):
+        ids = [report.id for report in self.reports[3:5]]
+        user = User.objects.get(username='DELETE_USER')
```
Comment (Jkrzy) [inline crt_portal/cts_forms/tests/test_forms.py:469]: Small opportunity here to use `self.user` instead of fetching from database again.

**Review by Jkrzy — CHANGES_REQUESTED** [review]
Needs a quick tweak to prevent Judicial district from being modified on form submission but looks great otherwise!

File: crt_portal/cts_forms/forms.py (hunk 1 above)
Comment (cantsin) [reply crt_portal/cts_forms/forms.py:1295]: Yes! That's right, disabled == read-only and thus browsers never submit this value. D'oh. Fixed.

**Review by Jkrzy — APPROVED** [review]
Looks good! I'd love to see `Judicial District` removed from the form altogether if the intention is for it to **not** be modifiable with a BulkAction.

cc: @randyabramson, @blaughman

**Pull request merged**